src/main/java/ch/uzh/ifi/hase/soprafs23/controller/UserController.java

```diff
@@ -104,23 +104,22 @@ public ResponseEntity<UserGetDTO> loginUser(@RequestBody UserPostDTO userPostDTO
     return ResponseEntity.status(HttpStatus.OK).headers(headers).body(userGetDTO);
   }
 
-  @PostMapping("users/{userId}/logout")
-  @ResponseStatus(HttpStatus.NO_CONTENT) //OK is 200
-  @ResponseBody
-  public void logoutUser(@PathVariable Long id, HttpServletRequest request){
-    // check if user is trying to log themselves out
-    Long tokenId = userService.getUserByToken(request.getHeader("X-Token"));
-    if(tokenId != id) {
-      throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "You are not authorized.");
+    @PostMapping("users/{userId}/logout")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Imo it would be helpful to keep the comments about what each http status means. like here add the // OK is 200 again.